tests/util/block.ts

```diff
@@ -210,7 +207,12 @@ export const verifyBlockFees = async (
                   // additional tip eventually paid by the user (maxPriorityFeePerGas) is purely a
                   // prioritization component: the EVM is not aware of it and thus not part of the
                   // weight cost of the extrinsic.
-                  gasPrice = BigInt((await context.web3.eth.getBlock(number - 1)).baseFeePerGas);
+
+                  // TODO: why did this break? ultimately we (currently) use our FixedGasPrice which
+                  // returns a constant 1GWEI (For moonbase). So are the block headers wrong now for
+                  // some reason?
+                  // gasPrice = BigInt((await context.web3.eth.getBlock(number - 1)).baseFeePerGas);
+                  gasPrice = 1_000_000_000n;
```

Comment:
@tgmichel Any idea what broke here?

Reply:
Fixed in https://github.com/PureStake/moonbeam/pull/1825/commits/570da7d46bfc5a49f88a5038a268c8027e99bdf7

Reply:
Thanks!